### PR TITLE
HASS OpenPlantbook integration  v.1.3

### PR DESCRIPTION
**Pull request by slaxor505**
Version 1.3-beta1
- Implemented Plant-sensors data uploader including config flow UI
- Added OPB API Credentials validation for Initial setup
- Added location sharing (Country and Coordinates)
- Removed OPB Tokens from Debug logs
- Replaced integration library with OpenPlantbook SDK
- Added logging for openplantbook_sdk
- Extended README.md documentation with new functionality
- Added service to manually trigger Uploading
- Added dependencies to manifest.json

### CODE REVIEW

File: custom_components/openplantbook/manifest.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
   "documentation": "https://github.com/Olen/home-assistant-openplantbook/",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/Olen/home-assistant-openplantbook/issues",
-  "requirements": ["pyopenplantbook==0.0.4"],
-  "version": "1.2.0"
+  "dependencies": ["recorder","history"],
+  "requirements": ["openplantbook-sdk==0.4.5","json-timeseries==0.1.4"],
+  "version": "1.3.0",
```
Comment (Olen) [inline custom_components/openplantbook/manifest.json:11]: Please keep the beta version tag for now.  Github actions uses this to auto create releases.

Comment (slaxor505) [reply custom_components/openplantbook/manifest.json]: Got it. Done in the latest commit.

**Comment by slaxor505** [comment]
Done. Reformatted.

**Comment by Olen** [comment]
Thanks.  I'll merge this and release a 1.3-beta.
If a few people verify that it works as expected, I'll release it as 1.3.

**Pull request merged**